### ISSUE TITLE
How do i create label_list exactly

### ISSUE THREAD

**Issue by sani1486** [bgshih/crnn#137]
Hello ,
I have been searching everywhere to know how to create the label_list file ?

lets say my Image_list file looks like 
image1.jpg
image2.jpg
image3.jpg 
so is it the proper way to create the imagepath_list file ? an

and also  if these 3  images have words "cat"  "dog"  "batman"  in it how should i create the label_list ?
cat
dog
batman

like this? or 

image1.jpg cat
image2.jpg dog
image3.jpg batman

like this ? 

can anybody please explain me this 2 question

Regards

**Comment by haneSier** [comment]
You can refer to this repository https://github.com/Sierkinhane/crnn_chinese_characters_rec maybe it will help you

**Comment by sani1486** [comment]
Yes , I have checked it . Thank you very much for the suggestion
